### PR TITLE
Add transaction-service API reference to DI

### PR DESCRIPTION
**Pull request by dhruvmisra**

### CODE REVIEW

File: api-references/data/insights.json [hunk 1]
```diff
@@ -1,1177 +1,2682 @@
...
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/responses.Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
```
Comment (aditya-67) [inline api-references/data/insights.json:813]: There is no property `definitions` or `securityDefinitions` in OpenAPI v3.

Screenshot 2024-09-04 at 09 41 49

**Review by aditya-67 — CHANGES_REQUESTED** [review]

File: api-references/data/insights.json [hunk 2]
```diff
@@ -1,1177 +1,2682 @@
...
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "title": "Response 200 Get Insights V1 Insight Get",
+                  "anyOf": [
```
Comment (aditya-67) [inline api-references/data/insights.json:226]: Any reason to use `anyOf` and not `oneOf`?

Comment (dhruvmisra) [reply api-references/data/insights.json:226]: I think this has been generated by the backend service. This API already exists on docs, I haven't modified it. Do our docs not support `anyOf` today?

File: api-references/data/insights.json (hunk 1 above)
Comment (dhruvmisra) [reply api-references/data/insights.json]: This has been fixed. Moved the new schemas to `components/schemas`.

File: api-references/data/insights.json (hunk 2 above)
Comment (aditya-67) [reply api-references/data/insights.json:222]: Understood. Currently, we do `allOf`, `oneOf`, `anyOf` should be similar, I'll get this working. Closing in favour of fixing for implementing `oneOf`.

**Review by aditya-67 — APPROVED** [review]

**Pull request merged**